app/controllers/course/users_base_controller.rb

```diff
@@ -0,0 +1,4 @@
+# frozen_string_literal: true
+class Course::UsersBaseController < Course::ComponentController
```

Comment:
This is very annoying, because we can't namespace properly.

@allenwq which is better? Course::UsersBaseController, CourseUser::Controller, Course::UserBaseController?